src/main/flight/filter_pt1.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * filter_pt1.c
+ *
+ *  Created on: 24 jun. 2015
+ *      Author: borisb
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <math.h>
+
+#include "common/axis.h"
+
+#include "flight/pid.h"
+#include "flight/filter_pt1.h"
+
+uint16_t cycleTime;
+
+// PT1 Low Pass filter
+float filter_pt1(float input, pt1_state_t* state, int f_cut) {
```

Comment:
Exactly what I was thinking, but changed my mind afterwards.
fitler.c should be used for future filter implementations to add different filters.